src/MapViewController.swift

```diff
@@ -17,7 +17,54 @@ import OnTheRoad
   case generalError, annotationDoesNotExist, apiKeyNotSet, routeDoesNotExist
 }
 
-open class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public protocol MapSingleTapGestureDelegate : class {
```

Comment:
Why limit the protocol to only be implemented by a class? Structs and enums are generally the "recommended" types in Swift as the value types are far safer to use.